src/styles/main.scss

```diff
@@ -1,3 +1,80 @@
-body {
-  margin: 0;
+$padding: 10px;
+$gap: 1px;
+$columns: 7;
+$cellSize: 100px;
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: flex-start;
+  padding: $padding;
+  max-width: ($cellSize + $gap) * ($columns + 1);
+  margin: 0 auto;
+  gap: $gap;
+
+  &__day {
+    width: $cellSize;
+    height: $cellSize;
+    background-color: #eee;
+    border: 1px solid black;
+    position: relative;
+    transition: all ease-in-out 0.3s;
```

Comment:
List all properties that you apply transition to. It will be easier to understand which properties are affected by it and also there will be no case in which something that should not have a transition has one